librepcb/controlpanel/controlpanel.cpp

```diff
@@ -294,9 +294,11 @@ void ControlPanel::projectEditorClosed() noexcept
 
 void ControlPanel::on_actionAbout_triggered()
 {
-    QMessageBox::about(this, tr("About"), tr(
-        "LibrePCB is a free & open source schematic/layout-editor."
-        "<br><br>"
+    QMessageBox::about(this, tr("About LibrePCB"), tr(
+        "<h1>About LibrePCB</h1>"
+        "<p>LibrePCB is a free & open source schematic/layout-editor.</p>"
+        "<p>Version: " GIT_VERSION "</p>"
+        "<p>Please see <a href='https://librepcb.org/'>librepcb.org</a> for more information.</p>"
```

Comment:
- Free certificates: https://startssl.com/
- 4$ certificates: https://gogetssl.com/domain-validation/domain-ssl/
- ...and many more.

Github pages doesn't support certificates on custom domains though. I could host the content & certificate for you, but then we wouldn't have auto-updating on push anymore.